src/main/java/com/fredsonchaves07/moviecatchapi/domain/useCases/user/CreateUserUseCase.java

```diff
@@ -56,15 +60,21 @@ private void validateUser() {
         if (emailAlreadyExist()) throw new EmailAlreadyExistException();
     }
 
+
     private boolean emailAlreadyExist() {
         return userRepository.findByEmail(user.getEmail()) != null;
     }
 
     private void saveUser() {
+        encryptPassword();
```

Comment:
Think of a way to put this method in another method. It might be interesting to put it in the post validation user creation method